common/app/views/fragments/nav/userAccountLinks.scala.html

```diff
@@ -0,0 +1,62 @@
+@()
+
+@import conf.Configuration
+@import conf.switches.Switches.{IdentityProfileNavigationSwitch, SaveForLaterSwitch}
+
+@if(IdentityProfileNavigationSwitch.isSwitchedOn) {
+
+    <li class="navigation-group__item navigation-group__item--user-account js-show-user-account-links">
+        @fragments.inlineSvg("profile-36", "icon", List("navigation-group__icon", "navigation-group__user-icon", "navigation-group__profile-icon"))
+
+        <div class="user-account__login">
+            <a href="@Configuration.id.url/signin?INTCMP=DOTCOM_HEADER_SIGNIN">
+                Sign in
+            </a>
+        </div>
+
+        <div class="user-account__signed-in">
+            <details>
+                <summary class="main-navigation__item__button user-account__signed-in__label"> my account </summary>
+
+                <ul class="user-account__signed-in__links">
+                    @if(SaveForLaterSwitch.isSwitchedOn) {
+                        <li class="user-account__signed-in__link">
+                            <a href="@Configuration.id.url/saved-for-later"
+                            data-link-name="Saved for Later">
+                                Saved for later
+                            </a>
+                        </li>
+                    }
+                    <li class="user-account__signed-in__link js-add-comment-activity-link" hidden>
```

Comment:
Given there isn't any logic in this template (woohoo!), I don't think it would add much to abstract the links, classes, data names, and display names for these things. They won't change much and are not editionalised. That said, if it makes it more readable we should abstract!